src/extras/synchronizer.js

```diff
@@ -192,13 +192,10 @@ function attachZoomHandlers(gs, syncOpts, prevCallbacks) {
         }
 
         block = true;
-        var opts = {
-          dateWindow: me.xAxisRange()
-        };
-        if (!me.isZoomed('x'))
-          opts.dateWindow = null;
-        if (syncOpts.range)
-          opts.valueRange = me.yAxisRange();
+        
+        var opts = {};
+        opts.dateWindow = me.isZoomed('x') ? me.xAxisRange() : null;
```

Comment:
Is this `null` assignment deliberate, or do you mean…

```
if (me.isZoomed('x'))
  opts.dateWindow = me.xAxisRange();
```

… instead?

Reply:
I don't know if this is necessary, it is working in my project without, but it just keeps previous code:
If graph is not zoomed, it assigns `null`. And I just added the same for `y` to make logic for both consistent

Reply:
It is just a little optimization actually. It doesn't call `xAxisRange()` now if it is just going to use `null`

Reply:
Ah right, that was in the original code. Thanks. (It’s late.)